debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/LcrEventHandler.java

```diff
@@ -318,16 +318,19 @@ private String resolveClobChunkValue(List<ChunkColumnValue> chunkValues) throws
         for (ChunkColumnValue chunkValue : chunkValues) {
             data.append(chunkValue.getColumnData().stringValue());
         }
-        return data.toString();
+        return data.length() == 0 ? null : data.toString();
     }
 
     private byte[] resolveBlobChunkValue(List<ChunkColumnValue> chunkValues) {
         long size = chunkValues.stream().map(ChunkColumnValue::getColumnData).mapToLong(Datum::getLength).sum();
```

Comment:
Unrelated to the change itself, but let's avoid the stream op here and use a simple for loop. Every bit matters in the event handling loop :)

Reply:
Fair enough, I'll push one extra commit here to adjust that.